client/src/test/resources/templates/item-content-with-atom-viewpoints.json

```diff
@@ -0,0 +1,72 @@
+          "labels": [],
+          "id": "4",
+          "data": {
+            "name": "Embed test 2",
+            "viewpoints": [
+              {
+                "commenter": {
+                  "name": " Jeb Bush",
+                  "imageUrl": "http://static.guim.co.uk/sys-images/Guardian/Pix/pictures/2016/1/12/1452598832981/JebBushR.png",
+                  "description": "Former Florida governor",
+                  "party": "Republican"
+                },
+                "quote": "If this works I'll be happy, over the moon in fact",
+                "date": "2016-01-14T23:34:00Z"
+              },
+              {
+                "commenter": {
+                  "name": "Hilary Clinton",
+                  "imageUrl": "http://static.guim.co.uk/sys-images/Guardian/Pix/pictures/2016/1/12/1452598832111/HillaryClintonR.png",
+                  "description": "Former secretary of state",
+                  "party": "Democrat"
+                },
+                "quote": "I'm all over this atoms stuff, not so hot on teamcity problems,",
+                "date": "2016-01-12T23:34:00Z"
```

Comment:
Is this really what is returned by the Content API?
A viewpoint atom may contain multiple viewpoints??